WalletWasabi.Gui/Controls/QrCode.cs

```diff
@@ -30,6 +31,35 @@ public QrCode()
 				.Where(x => x != null)
 				.ObserveOn(RxApp.MainThreadScheduler)
 				.Subscribe(x => FinalMatrix = AddPaddingToMatrix(x));
+
+			this.WhenAnyValue(x => x.QRImageSavePath)
+				.Where(x => !string.IsNullOrWhiteSpace(x) || !string.IsNullOrEmpty(x))
+				.Where(x => !(FinalMatrix is null))
```

Comment:
It's strange.

Reply:
Use `FinalMatrix != null`
That is the code convention AFAIK.

Reply:
I wish for a `is not` operator on C# vNext...

Reply:
@nopara73 @molnard @jmacato
In C# 8.0 there is this `if (variable is { })` which reads as if variable is something (not null) 
OR this `if (variable is object)`
https://www.youtube.com/watch?v=TJiLhRPgyq4&t=622s
https://intellitect.com/check-for-null-not-null/

Reply:
@yahiheb nice! We should start using one of it, since we're at it. Which seems to be the consensus?

Reply:
@nopara73 @yahiheb IMO `if (variable is object)` is a bit more obvious than the other one i think :)

Reply:
> it says:
This syntax works with C# 8.0’s static analysis so later code will know that variable has been checked for null.

Same for the other one.

> IMO `if (variable is object)` is a bit more obvious than the other one

 I agree, however there are more things you can do with the other syntax, take a look at the video.